liftoff_giftlist/src/main/java/org/launchcode/giftlist/models/Party.java

```diff
@@ -1,24 +1,31 @@
 package org.launchcode.giftlist.models;
 
 import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
 import java.util.ArrayList;
 import java.util.List;
 
 @Entity
 public class Party extends AbstractSuper{
 
+    @NotBlank
+    private String name;
+
+    private String description;
+
     @ManyToOne
     private User partyOwner;
 
-    @ManyToMany(mappedBy = "joinedParties")
+    @ManyToMany
     private List<User> members = new ArrayList<>();
 
     @ManyToMany
     private List<WishList> memberWishLists = new ArrayList<>();
```

Comment:
My thought all along is that within a given group/party, each member can only have one wish list connected to that group. This is to keep things simple, otherwise everyone else in the group may have to browse through multiple wishlists for each other person, making it unwieldy and non-user-friendly. That's my idea at least.
Thoughts? If we go that route, then when each user joins a group/party, they pick one of their wishlists and link it to that party.

Reply:
I have no idea why it linked my question to the old version. Weird...

Reply:
That is the current plan to set things up. Starting with the "Party Leader" (Whoever created the group, to keep things simple for now), when viewing the Groups page, there will be a way they can choose a wishlist to add to the group. If they already have a wishlist present on the page, they will not be able to "add another", I figured we could put a note with a link if they want to modify that specific list, that takes them to their wishlist page where they can modify that list.